numpy/testing/pytest_tools/utils.py

```diff
@@ -1219,16 +1219,7 @@ def assert_raises(exception_class, fn=None, *args, **kwargs):
     if fn is not None:
         pytest.raises(exception_class, fn, *args,**kwargs)
     else:
-        @contextlib.contextmanager
-        def assert_raises_context():
-            try:
-                yield
-            except BaseException as raised_exception:
-                assert isinstance(raised_exception, exception_class)
-            else:
-                raise ValueError('Function did not raise an exception')
-
-        return assert_raises_context()
+        return pytest.raises(exception_class)
```

Comment:
Maybe pass on `kwargs` too?

Reply:
Look at what it's replacing: no kwargs are present. The documentation specifies that there should be no kwargs when `fn` is none.

Reply:
Grep https://docs.pytest.org/en/latest/builtin.html for "context manager"

Reply:
Right, the documentation specifies that, but the implementation just silently drops them. Forwarding them would probably cause pytest to throw an appropriate error.

Reply:
Maybe I could simply remove the branch, though I seem to recall that previously failed.